test/controllers/preview_controller_test.exs

```diff
@@ -44,9 +25,7 @@ defmodule CodeCorps.PreviewControllerTest do
     end
```

Comment:
Is there a better way to refactor these lines, or should they be removed? What are we testing here?